src/cls/IPM/Storage/SystemRequirements.cls

```diff
@@ -41,10 +45,30 @@ Method CheckVersion() As %Status
 	}
 }
 
+/// Checks if Interoperability specification is satisfied. Health="true" implies Interoperability="enabled"
 Method CheckInteroperability() As %Status
 {
-	If (..Interoperability="enabled") && (('##class(%EnsembleMgr).IsEnsembleNamespace($Namespace)) || ('##class(%EnsembleMgr).validateNamespace($Namespace, 1))) {
-		Return $$$ERROR($$$GeneralError, "The module requires Interoperability Production Enabled. Namespace <"_$Namespace_"> doesn't support Interoperability Production")
+	If ..Interoperability = "enabled" {
+		Set tInteroperabilityRequired = 1
```

Comment:
This can be simplified to a simpler expression:
`Set interopRequired = $Case(..Interoperability, "enabled": 1, : 0) || ..Health`